buildkite/bazelci.py

```diff
@@ -2254,9 +2254,7 @@ def set_env_var(config_key, env_var_name):
     if not (
         is_pull_request()
         or use_but
-        or (os.getenv("BUILDKITE_BRANCH") != "master" 
-            and os.getenv("BUILDKITE_BRANCH") != "stable"
-            and os.getenv("BUILDKITE_BRANCH") != "main")
+        or os.getenv("BUILDKITE_BRANCH") not in ("master", "stable", "main")
```

Comment:
sorry for yet another late comment, but can you please get rid of the double negation?

Reply:
Done, applied de-Morgan laws :)

Reply:
Done on the comment as well.

Reply:
Nice, thanks!